content/en/docs/reference/glossary/cluster.md

```diff
@@ -4,20 +4,19 @@ id: cluster
 date: 2019-06-15
 full_link: 
 short_description: >
-   A set of worker machines, called nodes, that run containerized applications. Every cluster has at least one worker node.
-
+   A set of machines hosting containerized workloads, in the form of pods, and the control plane that manages them.  
 aka: 
 tags:
 - fundamental
 - operation
 ---
-A set of worker machines, called {{< glossary_tooltip text="nodes" term_id="node" >}},
-that run containerized applications. Every cluster has at least one worker node.
+A set of machines hosting containerized {{< glossary_tooltip text="workloads" term_id="workload" >}}, in the form of {{< glossary_tooltip text="pods" term_id="pod" >}}, and the {{< glossary_tooltip text="control plane" term_id="control-plane" >}} that manages them.  
 
 <!--more-->
-The worker node(s) host the {{< glossary_tooltip text="Pods" term_id="pod" >}} that are
-the components of the application workload. The
-{{< glossary_tooltip text="control plane" term_id="control-plane" >}} manages the worker
-nodes and the Pods in the cluster. In production environments, the control plane usually
-runs across multiple computers and a cluster usually runs multiple nodes, providing
-fault-tolerance and high availability.
+Machines may be physical or virtual.  Machines that host pods are referred to as {{< glossary_tooltip text="nodes" term_id="node" >}}.  Nodes that host workloads are referred to as worker nodes. There must be at least one worker node within a cluster. 
+
+Control plane components run on one or more of the machines within the cluster. They may execute directly in the context of the operating system of a machine, or within pods.  When control plane components run within pods on a machine, the machine may also be referred to as a control plane node.  
```

Comment:
No, this isn't correct. You can run the control plane within the cluster, but we actually recommend not doing that for the very best resilience.

Reply:
Control plane:
- always part of the cluster
- optional to run it on nodes
- if you run it on a separate server, the thing you run on a separate server is (if healthy) still logically a control plane, and that logical thing is part of the cluster

Reply:
So I'm having trouble with the distinction between "part of the cluster" and "within the cluster". :)

Reply:
Got it.  I could co-host my control plane instance, with a bunch of other control planes for other clusters, on the same machines.  I could host my control plane within another cluster that has it's own control plane.... etc.  In tuo capite fiat!   Will nudge this and other PRs to reflect.  Thx.

Reply:
Please do update this when you have time to.